app/controllers/application_controller.rb

```diff
@@ -1,10 +1,142 @@
-require './config/environment'
-
 class ApplicationController < Sinatra::Base
+set :views, Proc.new { File.join(root, "../views/") }
 
   configure do
-    set :public_folder, 'public'
-    set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "my_little_secret"
+  end
+
+    get '/' do
+
+      erb :index
+    end
+
+#------- Tweet Controls ------------#
+    get '/tweets' do
+      if Helpers.logged_in?(session)
```

Comment:
A `helpers` block is the convention in Sinatra -- bonus: you won't have to pass in the session!